src/main/java/org/breedinginsight/model/Column.java

```diff
@@ -46,4 +48,12 @@ public enum ColumnDataType {
         INTEGER,
         DOUBLE
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
"You must override hashCode() in every class that overrides equals(). Failure to do so will result in a violation of the general contract for Object.hashCode()"

see https://www.geeksforgeeks.org/override-equalsobject-hashcode-method/

Reply:
Thanks for catching this, fixed!